docs/user/science.rst

```diff
@@ -0,0 +1,142 @@
+
+    Build and publish your project for every change made through Git (GitHub, GitLab, BitBucket etc). Preview changes via Pull Requests. Receive notifications when something is wrong. How does this work? Have a look at this video:
+
+    .. TODO: Where should this video live?
+
+    .. video:: https://overtag.dk/files/enable-pull-request-builders.mp4
+       :width: 500
+       :height: 300
+
+.. dropdown:: 💬 Collaboration and community
+
+    .. figure:: /img/screenshot_edit_on_github.png
+        :align: right
+
+    Science and academia have a big kinship with software developers: We ❤️ community. Our solutions and projects become better when we foster inclusivity and active participation. Read the Docs features easy access for readers to suggest changes via GitHub, GitLab, BitBucket etc. But not just any unqualified feedback. Instead, the *code* and all the tools are available for your community to forge qualified contributions.
+
+    Your readers can become your co-authors!
+
+    Discuss changes via Pull Request and track all changes in your project's version history.
+
+    Does this sound chaotic? Don't worry, the full ownership and permission handling remains in your hands: Project and organization owners govern what is released and who has access to approve and build changes.
+
+.. dropdown:: 🔎 Search Engine Optimization
+
+    Don't reinvent SEO: Have SEO handled jointly by the combined best-practices from Sphinx, its themes and Read the Docs' platform.
+
+.. dropdown:: 🌱 Grow your own solutions
+
+    The eco-system is Open Source and makes it accessible for anyone with Python skills to build their own extensions.
+
+We want science communities to use Read the Docs and to be part of the documentation community 💞
+
+Getting started: Jupyter Book
+-----------------------------
+
+:external+jupyterbook:doc:`Jupyter Book <intro>` on Read the Docs brings you the rich experience of computated `Jupyter <https://jupyter.org/>`__ documents built together with a modern documentation tool. The results are beautiful and automatically deployed websites, built with Sphinx and :doc:`Executable Book <executablebook:index>` + all the extensions available in this ecosystem.
+
+Here are some popular activities that are well-supported by Jupyter Book:
+
+* Publications and books
+* Course and research websites
+* Interactive classroom activities
+* Data science software documentation
+
+
+.. button-ref:: executablebook:gallery
+    :ref-type: doc
+    :color: primary
+
+    Visit the gallery of solutions built with Jupyter Book
+
+Ready to get started?
+"""""""""""""""""""""
+
+.. Note that this is a deliberate repitition of a previous segment. Should it repeat? Maybe not, but for now it's nice to be sure that people see it.
+
+* All new to this? Take the official :external+jupyterbook:doc:`Jupyter Book Tutorial <start/your-first-book>`
+* Curious for practical code? See :doc:`/examples`
+* Familiar with Sphinx? See :doc:`/guides/jupyter`
```

Comment:
This might be a better section for a prominent button instead, if we have a singular place we want to send readers to continue.

Reply:
Since these are 3 alternative exit points, I don't want to give any of them more attention than the next. I don't think that the buttons will be all that nice in the list, and I think we might want to add some more "Getting started" resources here over time, as the audience is very non-homogeneous.

Reply:
I've tested with buttons and I can't get it to work. The sphinx-design buttons are block level elements and won't look good inline, even if we CSS'ed them.

There isn't really a design element with buttons that fits what we want here: Multiple resources listed with clearly visible links.

I've adapted the page slightly now, I think we should work with how anonymous the text is becoming in a future version :)

![image](https://user-images.githubusercontent.com/374612/183679469-7596639a-8beb-4277-9557-1824750ada5e.png)